### PR TITLE
Incorporating unit tests for environment handler

### PR DESCRIPTION
**Pull request by SiniWilson**
### Requirements

* Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.
* All new code requires tests to ensure against regressions

### CODE REVIEW

File: core/jazz_environment-event-handler/test/test.js [hunk 1]
```diff
@@ -15,642 +15,1496 @@
...
+      body: { "message": "unautho" }
+    };
+
+    let authStub = sinon.stub(rp, 'Request').returns(Promise.resolve(responseObject));
+    requestPromiseStub.callsFake((obj) => {
+      return obj.callback(null, responseObject, responseObject.body);
+    });
+    let message = 'User is not authorized to access this service';
+    index.handler(event, context, (err, res) => {
+      if (err) {
```
Comment (svidhani) [inline core/jazz_environment-event-handler/test/test.js]: try not to use if and else in tests. Ensure your test is set up to either get err or data. You test should be deterministic. Split this and make 2 tests. One which surely gives and error and another with data(success). Please update other tests if you have taken similar approach

File: core/jazz_environment-event-handler/test/test.js [hunk 2]
```diff
@@ -15,642 +15,1496 @@
...
+    };
+    let service_responseObject = {
+      statusCode: 200,
+      body: JSON.stringify(body)
+    };
+    processRequestStub.resolves(service_responseObject.body);
+    getServiceDetailsStub.resolves(service_responseObject.body);
+    requestPromiseStub.callsFake((obj) => {
+      if (obj.uri == "https://{conf-apikey}.execute-api.{conf-region}.amazonaws.com/prod/jazz/services?service=test-env-oss-3&domain=jazztesting&isAdmin=true") {
+        return obj.callback(null, service_responseObject, service_responseObject.bod
```
Comment (svidhani) [inline core/jazz_environment-event-handler/test/test.js]: new line

File: core/jazz_environment-event-handler/test/test.js (hunk 1 above)
Comment (SiniWilson) [reply core/jazz_environment-event-handler/test/test.js]: Done

File: core/jazz_environment-event-handler/test/test.js (hunk 2 above)
Comment (SiniWilson) [reply core/jazz_environment-event-handler/test/test.js]: Done

**Pull request merged**